src/mantarray_desktop_app/main.py

```diff
@@ -68,7 +68,9 @@ def get_server_port_number() -> int:
     return server_manager.get_port_number()
 
 
-def _set_up_socketio_handlers(ws_queue: LightQueue) -> Callable[[], None]:
+def _set_up_socketio_handlers(
+    ws_queue: LightQueue, to_pm_queue: queue.Queue[Dict[str, Any]]
```

Comment:
Like I mentioned earlier, the naming convention is always `to/from main` never `to/from pm`, so we'll need to rename this queue. It also makes sense to rename the other queue to `to_ws_queue` now so it's very clear what each queue is for